src/pages/new-claim/new-claim.tsx

```diff
@@ -0,0 +1,140 @@
+import { FormEventHandler, useState } from 'react';
+import { useParams } from 'react-router';
+import { BaseLayout } from '../../components/layout';
+import Input from '../../components/input';
+import Button from '../../components/button';
+import { isEmpty } from 'lodash';
+import { Policy } from '../../chain-data';
+import { useClaimsManager } from '../../contracts';
+import { useUserPolicyById } from '../../logic/policies';
+import { formatApi3, parseApi3 } from '../../utils';
+import globalStyles from '../../styles/global-styles.module.scss';
+import styles from './new-claim.module.scss';
+
+interface Params {
+  policyId: string;
+}
+
+interface FormState {
+  evidence: string;
+  amount: string;
+}
+
+function getValidationMessages(form: FormState, policy: Policy) {
+  const messages: ValidationMessages<FormState> = {};
+
+  if (form.evidence.trim().length === 0) {
+    messages.evidence = 'Please fill in this field';
+  }
+
+  // Don't try to parse if the user is starting to type a negative number
+  if (form.amount.replace(/-/g, '').length === 0) {
+    messages.amount = 'Please fill in this field';
```

Comment:
Nit: This message is a bit inaccurate.

I'd suggest using `goSync` and inside of it checking for the expected errors (positive number, <= coverage amount) and saying generic `Please enter a valid number` in case it fails.

This should wokr:
```
  const goAmountValidation = goSync(() => {
    const parsed = parseApi3(form.amount);

    if (parsed.lte(0)) return 'This must be greater than zero';
    if (parsed.gt(policy.coverageAmount)) return 'This must not exceed the coverage amount';
  });
  if (!goAmountValidation.success) messages.amount = 'Please enter a valid number';
  else if (goAmountValidation.data) messages.amount = goAmountValidation.data;

  return messages;
```